examples/next/pages/ui/components/ai/ai-conversation/custom-actions/index.page.tsx

```diff
@@ -0,0 +1,48 @@
+import { MdCopyAll } from 'react-icons/md';
+import { AIConversation } from '@aws-amplify/ui-react-ai';
+import '@aws-amplify/ui-react/styles.css';
+import '@aws-amplify/ui-react-ai/ai-conversation-styles.css';
+
+import { Authenticator, Card } from '@aws-amplify/ui-react';
+import { INITIAL_MESSAGES } from '../../constants';
```

Comment:
For a lot of the UI customization stuff, we don't actually need a backend to be deployed, we can just pass in static messages to test. Neat!